pig/src/main/java/org/jboss/pnc/bacon/pig/PigFacade.java

```diff
@@ -161,29 +165,45 @@ public static GroupBuildInfo run(
         // TODO: should we bother with this case?
 
         RepositoryData repo = null;
-
-        if (repoZipPath != null) {
-            repo = parseRepository(new File(repoZipPath));
+        
+        if (repoZipPath != null || context.getPigConfiguration()
```

Comment:
if strategy is ignore and there is not repoZip specified, we're not generating repo